service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/bucket/CopyGcsBucketDefinitionStep.java

```diff
@@ -62,85 +61,71 @@ public CopyGcsBucketDefinitionStep(
   @Override
   public StepResult doStep(FlightContext flightContext)
       throws InterruptedException, RetryException {
-    final FlightMap inputParameters = flightContext.getInputParameters();
-    final FlightMap workingMap = flightContext.getWorkingMap();
+    FlightMap inputParameters = flightContext.getInputParameters();
+    FlightMap workingMap = flightContext.getWorkingMap();
     FlightUtils.validateRequiredEntries(
         inputParameters,
         ControlledResourceKeys.DESTINATION_WORKSPACE_ID,
         ControlledResourceKeys.DESTINATION_RESOURCE_ID);
-    final String resourceName =
+    var resourceName =
```

Comment:
The rule for `var` is that we only use it if the type is obvious from the right-hand-side expression.
In this case, that is not the case. These should specify their types.

Reply:
i 100% agree with you that we should only allows var when the type is obvious. In this case, the type is pretty obvious as `getInputParameterOrWorkingValue` always specified type https://cs.github.com/DataBiosphere/terra-workspace-manager/blob/9629c9fb57fd60b0ace6398fb91f117805dc8879/service/src/main/java/bio/terra/workspace/common/utils/FlightUtils.java?q=getInputParameterOrWorkingValue#L82.

Reply:
FWIW I agree with Dan. There's been times where I see `var` and I don't know what the type is. For this case, one has to be familiar with `getInputParameterOrWorkingValue` to know that it's a String.

Reply:
specified all the ones that are from custom methods.